### PR TITLE
Update Package.swift

### PR DESCRIPTION
**Pull request by danibachar**
Bumping to support macv13

### CODE REVIEW

**Comment by danibachar** [comment]
Currently the package does not build because its pointing to [SPM](https://github.com/apple/swift-package-collection-generator) main - which is not supporting macos_v12

File: Package.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "1.2.2")),
-        .package(url: "https://github.com/apple/swift-package-manager.git", branch: "main"),
+        .package(url: "https://github.com/apple/swift-package-manager.git", revision: "82e3974c7714783bab1da11e29d053422fdfba9a"), // 5.10 Release
```
Comment (yim-lee) [inline Package.swift:16]: I created a new branch `5.10` in this repo that points to SwiftPM's `release/5.10` branch. Let's keep the `branch: "main"` here.

**Review by yim-lee — CHANGES_REQUESTED** [review]
Thanks for the PR. I think we only need the change in `platforms` to fix `main`.

**Comment by danibachar** [comment]
Thanks for the quick response, you are correct only the platform change is needed - i reverted the branch change.

**Review by yim-lee — APPROVED** [review]

**Pull request merged**